diff --git a/node_modules/three/build/three.js b/node_modules/three/build/three.js
index 4c47e91..e153046 100644
--- a/node_modules/three/build/three.js
+++ b/node_modules/three/build/three.js
@@ -12555,9 +12555,9 @@
 			get: function (name) {
 				const extension = getExtension(name);
 
-				if (extension === null) {
-					console.warn('THREE.WebGLRenderer: ' + name + ' extension not supported.');
-				}
+				// if (extension === null) {
+					// console.warn('THREE.WebGLRenderer: ' + name + ' extension not supported.');
+				// }
 
 				return extension;
 			}
@@ -17041,11 +17041,11 @@
 			if (source.version !== source.__currentVersion || forceUpload === true) {
 				_gl.pixelStorei(_gl.UNPACK_FLIP_Y_WEBGL, texture.flipY);
 
-				_gl.pixelStorei(_gl.UNPACK_PREMULTIPLY_ALPHA_WEBGL, texture.premultiplyAlpha);
+				// _gl.pixelStorei(_gl.UNPACK_PREMULTIPLY_ALPHA_WEBGL, texture.premultiplyAlpha);
 
-				_gl.pixelStorei(_gl.UNPACK_ALIGNMENT, texture.unpackAlignment);
+				// _gl.pixelStorei(_gl.UNPACK_ALIGNMENT, texture.unpackAlignment);
 
-				_gl.pixelStorei(_gl.UNPACK_COLORSPACE_CONVERSION_WEBGL, _gl.NONE);
+				// _gl.pixelStorei(_gl.UNPACK_COLORSPACE_CONVERSION_WEBGL, _gl.NONE);
 
 				const needsPowerOfTwo = textureNeedsPowerOfTwo(texture) && isPowerOfTwo$1(texture.image) === false;
 				let image = resizeImage(texture.image, needsPowerOfTwo, false, maxTextureSize);
@@ -17251,11 +17251,11 @@
 			if (source.version !== source.__currentVersion || forceUpload === true) {
 				_gl.pixelStorei(_gl.UNPACK_FLIP_Y_WEBGL, texture.flipY);
 
-				_gl.pixelStorei(_gl.UNPACK_PREMULTIPLY_ALPHA_WEBGL, texture.premultiplyAlpha);
+				// _gl.pixelStorei(_gl.UNPACK_PREMULTIPLY_ALPHA_WEBGL, texture.premultiplyAlpha);
 
-				_gl.pixelStorei(_gl.UNPACK_ALIGNMENT, texture.unpackAlignment);
+				// _gl.pixelStorei(_gl.UNPACK_ALIGNMENT, texture.unpackAlignment);
 
-				_gl.pixelStorei(_gl.UNPACK_COLORSPACE_CONVERSION_WEBGL, _gl.NONE);
+				// _gl.pixelStorei(_gl.UNPACK_COLORSPACE_CONVERSION_WEBGL, _gl.NONE);
 
 				const isCompressed = texture.isCompressedTexture || texture.image[0].isCompressedTexture;
 				const isDataTexture = texture.image[0] && texture.image[0].isDataTexture;
